composer/src/main/kotlin/com/gojuno/composer/TestRun.kt

```diff
@@ -181,7 +181,21 @@ private fun pullTestFiles(adbDevice: AdbDevice, test: InstrumentationTest, outpu
             )
         }
 
-private fun saveLogcat(adbDevice: AdbDevice, logsDir: File): Observable<Pair<String, String>> = Observable
+class LogLineParser {
+    fun parseTestClassAndName(logLine: String): Pair<String, String>? {
```

Comment:
I honestly like this non-regexp version better, much more understandable!

Reply:
I have a bug, I forgot the "newer" logs have a ton of stuff before "TestRunner". I'll submit an update later.

Reply:
Sure, so far it looks really good, keep it up 👍